projects/valtimo/dossier/src/lib/components/dossier-process-start-modal/dossier-process-start-modal.component.html

```diff
@@ -25,6 +25,16 @@
       (formFlowComplete)="formFlowSubmitted()"
     ></valtimo-form-flow>
   </div>
+  <div body *ngIf="isFormViewModel">
```

Comment:
If formDefinition is null then this will still be displayed, check the conditions